src/app/screens/App/screens/Home/components/Book/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { BOOKCOVERALTTEXT } from '../../strings';
+import './styles.css';
+import defaultBookImage from '../../../../../../../assets/ASSETS/default_book.svg';
+
+export default function Book(props) {
+  const has_image = props.book.image_url;
+  const image_url = props.book.image_url || defaultBookImage;
+
+  return (
+    <div key={props.book.id} className="book">
+      <div
+        className={
+          'book-image-container ' + (has_image ? '' : 'backgroundgray')
```

Comment:
dont use sting concatenation ( + ).
Do this:
\`book-image-container ${has_image ? '' : 'backgroundgray'}\`